cmd/config/init/rollapp.go

```diff
@@ -48,6 +48,9 @@ func initializeRollappConfig(initConfig config.RollappConfig) error {
 	if err != nil {
 		return err
 	}
+	if err = setRollappConfig(initConfig); err != nil {
```

Comment:
bad name, same as above func

 `setRollappAppConfig` probably should call
 `SetDefaultDymintConfig`,`setAppConfig` and `setTMConfig